test/integration/targets/vcenter_folder/aliases

```diff
@@ -1,2 +1,3 @@
 shippable/vcenter/group1
 cloud/vcenter
+vmware/scenario/vcenter_only
```

Comment:
How are you planning on using this alias?

Reply:
I'm working on Zuul CI job that just starts a vcenter. Only a subset of the test roles work fine in this cas. I would like to use the aliase to easily identify them.

Reply:
In that case it seems more appropriate to use something like `zuul/vcenter/group1` so it's more clearly associated with the Zuul CI setup you're working on.

Reply:
There is actually no dependency with Zuul. prepare_vmware_tests can deal with two topologies, 1vcenter or 1vcenter+2esxi, and this will be useful for anyone who wants to run the tests locally. This is the reason why I would prefer to avoid the zuul prefix.

Reply:
The reason for the `zuul` prefix is to indicate that Zuul depends on that group of tests and is responsible for running them -- which is the same reason we have `shippable` prefixed tests.

Users can run any group of tests they want locally, but the only groups we have defined currently are for use in CI. The prefixes make them self-documenting as to which system depends on them.

Reply:
Are you ok with `zuul/vmware/vcenter_only`? It comes with the zuul prefix and keep a meaning full name.

Reply:
That works for me, as long as all the tests in the group will be run together on Zuul. If they're going to be split into groups for parallel execution, then a group indicator of some kind should be included in the alias, like `zuul/vmware/vcenter_only/group1`.